entity-types/ext-triton/golden_metrics.yml

```diff
@@ -0,0 +1,48 @@
+totalCpuMemory:
+  title: Provides the total CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_total_bytes)
+    from: Metric
+
+usedCpuMemory:
+  title: Provides the used CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_used_bytes)
+    from: Metric
+
+cpuUtilization:
+  title: Provides the CPU utilization from the Triton inference server.
+  unit: PERCENTAGE
+  query:
+    select: latest(nv_cpu_utilization)
+    from: Metric
+
+nv_pinned_memory_pool_total_bytes:
```

Comment:
The same camelCase change need to be applied to all

Reply:
Hi @lujop,
As per your suggestions, I have made changes. Could you please review it.